### ISSUE TITLE
Create a landing page for gcloud-java

### ISSUE THREAD

**Issue by aozarov** [GoogleCloudPlatform/gcloud-java#66]
And add a landing page for  http://googlecloudplatform.github.io/gcloud-java and add a link to it from the other Veneer landing pages (in the existing top-left drop-down).

**Comment by aozarov** [comment]
The look and feel for  http://googlecloudplatform.github.io/gcloud-java landing page should match other Venneer pages and should be included in their drop down (looks like drop-down is missing in some like Python) and should have its own drop-down. 

Also, main page http://googlecloudplatform.github.io/ should have references to gcloud-libraries in search and in the language specific pages (looks like one is missing for Node.js).

**Comment by ludoch** [comment]
API documentation is not really an enhancement IMO. No doc = no product. Bad doc = bad press and no usage.
I hope the gcloud * APIS documentation is seen as a critical topic.

**Comment by jgeewax** [comment]
@blowmage @ryanseys Any chance you guys can point us in the right direction of how we'd pull this together ? Any CSS files floating around we should copy/paste ?

**Comment by blowmage** [comment]
@jgeewax I just shared a zip file with you on Google Drive that contains template files for gcloud language sites.

**Comment by ajkannan** [comment]
Does anybody know where to find a logo for Java comparable to the Node.js, Ruby, and Python .svg logos provided in the template?

**Comment by ludoch** [comment]
For now, let's not use an external image.

**Comment by blowmage** [comment]
I don't have a great way to share it other than inlining it here, but here is the java logo in SVG similar to the node and ruby and python images:

```xml
<?xml version="1.0" encoding="utf-8"?>
<!-- Generator: Adobe Illustrator 14.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 43363)  -->
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<svg version="1.1" id="java" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
  width="24px" height="24px" viewBox="0 0 24 24" enable-background="new 0 0 24 24" xml:space="preserve">
<path d="M8.851,18.561c0,0-0.918,0.533,0.652,0.714c1.901,0.217,2.873,0.186,4.97-0.211c0,0,0.551,0.345,1.32,0.645
  C11.095,21.722,5.16,19.591,8.851,18.561"/>
<path d="M8.276,15.933c0,0-1.028,0.761,0.542,0.924c2.032,0.21,3.637,0.227,6.414-0.308c0,0,0.384,0.39,0.987,0.603
  C10.539,18.812,4.211,17.282,8.276,15.933"/>
<path d="M13.116,11.475c1.158,1.334-0.304,2.532-0.304,2.532s2.94-1.516,1.59-3.417c-1.261-1.772-2.229-2.652,3.007-5.688
  C17.409,4.902,9.192,6.954,13.116,11.475"/>
<path d="M19.33,20.504c0,0,0.68,0.559-0.747,0.991c-2.712,0.822-11.288,1.07-13.669,0.033c-0.857-0.372,0.749-0.89,1.254-0.998
  c0.527-0.114,0.828-0.093,0.828-0.093c-0.952-0.671-6.155,1.317-2.643,1.887C13.933,23.878,21.815,21.625,19.33,20.504"/>
<path d="M9.292,13.21c0,0-4.362,1.037-1.545,1.412c1.189,0.159,3.562,0.123,5.77-0.062c1.806-0.151,3.618-0.477,3.618-0.477
  s-0.637,0.273-1.098,0.588c-4.429,1.165-12.987,0.623-10.523-0.568C7.597,13.096,9.292,13.21,9.292,13.21"/>
<path d="M17.116,17.584c4.503-2.34,2.422-4.588,0.968-4.285c-0.355,0.073-0.515,0.139-0.515,0.139s0.132-0.207,0.385-0.298
  c2.875-1.009,5.086,2.98-0.928,4.562C17.026,17.702,17.096,17.64,17.116,17.584"/>
<path d="M14.402,0c0,0,2.493,2.495-2.367,6.331c-3.896,3.077-0.888,4.832-0.001,6.835C9.76,11.115,8.091,9.308,9.21,7.626
  C10.855,5.16,15.408,3.962,14.402,0"/>
<path d="M9.734,23.925c4.323,0.276,10.959-0.154,11.117-2.199c0,0-0.302,0.775-3.572,1.392C13.59,23.811,9.04,23.73,6.341,23.284
  C6.341,23.284,6.893,23.742,9.734,23.925"/>
</svg>
```

**Comment by ryanseys** [comment]
gist.github.com ? :stuck_out_tongue_winking_eye:

**Comment by blowmage** [comment]
Either way it's the text of the SVG and not a file to download. :)

**Comment by ryanseys** [comment]
Gist gives you a little preview of it if you name the file with svg extension :) https://gist.github.com/ryanseys/3ae962fe1c87d5b0cb02

**Comment by ajkannan** [comment]
Issue fixed by merge #125
